packages/cactus-plugin-ledger-connector-besu/src/test/typescript/integration/plugin-ledger-connector-besu/deploy-contract/testLedgerParameters.ts

```diff
@@ -0,0 +1,115 @@
+    await besuTestLedger.start();
+
+    assert.tearDown(async () => {
+        await besuTestLedger.stop();
+        await besuTestLedger.destroy();
+    });
+    assert.equal(besuTestLedger.envVars,rinkebyNetworkEnvVars);
+    assert.ok(besuTestLedger);
+    assert.end();
+});
+
+tap.test("deploys a Besu Node on the Ropsten network", async (assert: any) => {
+    assert.plan(2);
+    // const rinkebyNetworkParameters = "--mount type=bind,source=/<myvolume/besu/testnode>,target=/var/lib/besu hyperledger/besu:latest --miner-enabled --miner-coinbase fe3b557e8fb62b89f4916b721be55ceb828dbd73--network=dev --min-gas-price=0";
+    const rinkebyNetworkEnvVars = ["BESU_MOUNT_TYPE=bind","BESU_MINER_ENABLED","BESU_MINER_COINBASE=fe3b557e8fb62b89f4916b721be55ceb828dbd73","BESU_SOURCE=/<myvolume/besu/testnode>","BESU_NETWORK=ropsten", "BESU_MIN_GAS_PRICE=0", "BESU_TARGET=/var/lib/besu hyperledger/besu:latest"];
+    const besuOptions = {
+        envVars: rinkebyNetworkEnvVars
+    }
+
+    const besuTestLedger = new BesuTestLedger(besuOptions);
+    await besuTestLedger.start();
+
+    assert.tearDown(async () => {
+        await besuTestLedger.stop();
+        await besuTestLedger.destroy();
+    });
+    assert.equal(besuTestLedger.envVars,rinkebyNetworkEnvVars);
+    assert.ok(besuTestLedger);
+    assert.end();
+});
+
+tap.test("deploys a Besu Node on the Goerli network", async (assert: any) => {
+    assert.plan(2);
+    // const rinkebyNetworkParameters = "--mount type=bind,source=/<myvolume/besu/testnode>,target=/var/lib/besu hyperledger/besu:latest --miner-enabled --miner-coinbase fe3b557e8fb62b89f4916b721be55ceb828dbd73--network=dev --min-gas-price=0";
+    const rinkebyNetworkEnvVars = ["BESU_MOUNT_TYPE=bind","BESU_MINER_ENABLED","BESU_MINER_COINBASE=fe3b557e8fb62b89f4916b721be55ceb828dbd73","BESU_SOURCE=/<myvolume/besu/testnode>","BESU_NETWORK=goerli", "BESU_MIN_GAS_PRICE=0", "BESU_TARGET=/var/lib/besu hyperledger/besu:latest"];
+    const besuOptions = {
+        envVars: rinkebyNetworkEnvVars
+    }
+
+    const besuTestLedger = new BesuTestLedger(besuOptions);
+    await besuTestLedger.start();
+
+    assert.tearDown(async () => {
+        await besuTestLedger.stop();
+        await besuTestLedger.destroy();
+    });
+    assert.equal(besuTestLedger.envVars,rinkebyNetworkEnvVars);
+    assert.ok(besuTestLedger);
+    assert.end();
+});
+
+
+tap.test("deploys a Besu Node on the Ethereum main network", async (assert: any) => {
+    assert.plan(2);
+    const ethereumEnvVars = ["BESU_TARGET=/var/lib/besu", "BESU_PORT=30303:30303", "BESU_RCP_HTTP_ENABLED"];
+    const besuOptions = {
+        envVars: ethereumEnvVars
+    }
+    const besuTestLedger = new BesuTestLedger(besuOptions);
+    await besuTestLedger.start();
+
+    assert.tearDown(async () => {
+        await besuTestLedger.stop();
+        await besuTestLedger.destroy();
+    });
+    assert.equal(besuTestLedger.envVars,ethereumEnvVars);
```

Comment:
@RafaelAPB 
This assertion might not need the backing container to be launched because the class constructor already assigns the `envVars` property regardless of `.start()` having been called or not.

Would it make sense to either (?)

a) Remove the `.start()`, `stop()` and `destroy()` calls (which speeds up the test cases 1000x fold but asserts the same thing) OR
b) Add assertions that verify that the env vars are also present within the container itself and are indeed in effect (I've encountered a lot of bugs in the past where I thought I set some configuration up for some software and later it turned out to be non-effective because of some bug in between).